README.md

```diff
@@ -1,78 +1,157 @@
+
+```
+### docker execute
+```bash
+docker exec -ti pathogenecity_anno bash
+```
+
+### cd to AutoGVP directory
+```bash
+cd /home/rstudio/AutoGVP
+```
+
+## How to Run
+
+### CAVATICA input ###
+1. Run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
+This workflow currently annotates variants with ClinVar (2022-05-07).
+2. Run the [Pathogenicity Preprocessing Workflow](https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing), which performs ANNOVAR with InterVar and AutoPVS1 annotations.
+3. Run AutoGVP.
+
+AutoGVP Requirements (recommended to place all in the `input/` folder):
+- VEP-, ANNOVAR-, and ClinVar annotated VCF file (`*VEP.vcf`)
+- ANNOVAR multianno file (`*hg38_multianno.txt`)
+- InterVar file (`*intervar.hg38_multianno.txt.intervar`)
+- AutoPVS1 file (`*autopvs1.txt`)
+- `variant_summary.txt`
+- `submission_summary.txt.gz` (can retrieve with `download_db_files.sh`)
+Note: the variant_summary and submission_summary files need to be uncompressed.
+
+Example run:
+
+```bash
+bash download_db_files.sh
+```
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf <*.vcf> --multianno <*multianno.txt> --intervar <*hg38_multianno.txt.intervar> --autopvs1 <*autopvs1.txt --output <output_prefex> --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
+```
+
+### Custom (non-CAVATICA) input ###
+1. Annotate the germline VCF with VEP.
+2. Run ANNOVAR with InterVar with the following options (to create the VCF input for AutoGVP).
+```perl
+perl table_annovar.pl input/test_hg38_selected_VEP_annotated.vcf hg38 --buildver hg38 --out test_hg38_selected --remove --protocol gnomad211_exome,gnomad211_genome --operation f,f --vcfinput
+```
+3. Run AutoPVS1.
+4. Optional: VCF annotation with ClinVar. If not done by the user, ClinVar file is required input to AutoGVP.
+4. Run AutoGVP.
+
+AutoGVP Requirements (recommended to place all in the `input/` folder):
+- VEP- and ANNOVAR-annotated VCF (`*VEP.vcf`)
+- ANNOVAR file (`*hg38_multianno.txt`)
+- InterVar file (`*intervar.hg38_multianno.txt.intervar`)
+- AutoPVS1 file (`*autopvs1.txt`)
+- ClinVar VCF (`clinvar_yyyymmdd.vcf.gz`; optional)
+- `variant_summary.txt`
+- `submission_summary.txt.gz` (can retrieve with `download_db_files.sh`)
+Note: the variant_summary and submission_summary files need to be uncompressed.
+
+Example run:
+```bash
+bash download_db_files.sh
+```
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf input/testing_010423_VEP.vcf --multianno input/testing_010423.hg38_multianno.txt --intervar input/testing_010423_intervar.hg38_multianno.txt.intervar --autopvs1 input/testing_010423_autopvs1.txt --clinvar input/clinvar_20211225.vcf.gz --output SRRT0182 --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
+```
+
+### Step by step workflow
+1. Read in ClinVar file
+2. Create `vcf_id` (chr and position)
```

Comment:
I thought vcf_id was chr_pos_ref_alt (not just chr and position)

Reply:
(i didn't write this, just copied it over) - you are right:

https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/895f6a5787c4cb7fc292222edf4e3e6761c2415b/AutoGVP/01-annotate_variants_custom_input.R#L96

will update